packages/pocl-job/README.md

```diff
@@ -0,0 +1,20 @@
+# Rod Licensing - Post Office Counter Licence Sales Processor
+
+The pocl-job package handles processing of Post Office Counter Licence sales files
+
+# Environment variables
+
+| name                      | description                                              | required | default   | valid                         | notes |
+| ------------------------- | -------------------------------------------------------- | :------: | --------- | ----------------------------- | ----- |
+| NODE_ENV                  | Node environment                                         |    no    |           | development, test, production |       |
+| POCL_FILE_STAGING_TABLE   | The DynamoDB table used for staging POCL files           |   yes    |           |                               |       |
+| POCL_RECORD_STAGING_TABLE | The DynamoDB table used for staging POCL records         |   yes    |           |                               |       |
+| POCL_STAGING_TTL          | The time to live for records in the either staging table |    no    | 168 hours |                               |       |
+
+###See also:
+
+- Environment variables [required by the connectors-lib package](../connectors-lib/README.md).
+
+# Prerequisites
+
+See [main project documentation](../../README.md).
```

Comment:
Can we have a description of the process. I would like to see a description the cli and an outline of the overall process. Is the transform to file supplementary?  I am presuming the JSON gets written directly into dynamo without an intermediate file

Reply:
Yes, agreed.  I am doing further work on it at the moment to add the SFTP/S3 support so will include documentation at this point.